src/test/data/tx_valid.json

```diff
@@ -512,5 +512,40 @@
+"020005000134adb250ae44db88313261bafa093e0b5cf70efab65c083929ce95d58ac4257e00000000fd08014c82473045022100b88bd6cd6407b6e608bc6b10a2b8395441aa31da1efbfa9021b512c60dd3c2470220641790ee0cc9b2cb2f82abe3c279498f2ff9e01f694ca200561912d69daa02b3141df04e02165c13d2bb05eec205360563e5b9dbfa205b7881d9dc1fa636463beee0dbc1e09f8ec219375d57ce0f57959aba08c10db9010501064c82473045022100977409fe64359e971a433f77a74de9555f37e10f1c8c258b9066f78eef57931d0220588c6ffb23f5cb80f5d7a04a92db9ca2042ffd5f9aa26e0ae0f15b329025aa4d141df04e02165c13d2bb05eec205360563e5b9dbfa20c73431cf49cde02e4b55956b142fc08beebd19428fc8e27c7aa51acfad41b2fb01050106ffffffff01005cb2ec22000000016a00000000", "P2SH,WITNESS"],
+
+["Withdraw transaction"],
+[[["e207854f1d1458388f297c257842c3c1d25016e5903b8905e38991e213f7d5c2", 0, "0x21 0x03d9aef87064c9e05d8123b8275c9c2d9508e46ee046dc5934dc91227764edaea7 OP_CHECKCOMMIT OP_IF 1 OP_ELSE OP_DUP OP_HASH160 0x14 0xbf4b80d8b0e5f5d6bf66cbb7c9cf15747f6c1c93 OP_EQUALVERIFY OP_CHECKSIG OP_ENDIF"]],
+"0200060001c2d5f713e29189e305893b90e51650d2c1c34278257c298f3858141d4f8507e2000000006a4730440220683071c3013d77865b32da54e63a0c2878b7c1f9ca20a2c7611c11bbeb42947e02204846e490bad92a27c7cd3a1e228eb7208a1c65c208c3324df939f41b9b8329b0012103d9aef87064c9e05d8123b8275c9c2d9508e46ee046dc5934dc91227764edaea7ffffffff01ac4bb2ec220000001976a914bf4b80d8b0e5f5d6bf66cbb7c9cf15747f6c1c9388ac00000000", "P2SH,WITNESS"],
+
+["Admin transaction"],
+[[["140fc01dffba984970a212d5569b2b281dcc1b54f4bb3dd3371ae4eb96f9bd51", 1, "OP_HASH160 0x14 0x421541ef76e3ebec18320e5622a8997ea3848b1c OP_EQUAL", 100000000]],
+"0200070000010151bdf996ebe41a37d33dbbf4541bcc1d282b9b56d512a2704998baff1dc00f140100000023220020371b504fe047484b6c95234d698ca4b5fa30a9fcbed5ae240bccc93ee06943f2ffffffff02405489000000000017a914421541ef76e3ebec18320e5622a8997ea3848b1c8700000000000000006b6a4c68020321026c350263dce5662334307f3eb488c89ae07253252af7093bc7d2788af03a220421021f8309dfa3feaf75d8d3fb63a90777823aff30758d2aacf3b8844113ab7145b42102f8dbb25cf6ba17a955379b1dfd9384f293dd779f7a5b337397c9b127ca31433c04004730440220333f14cf294b12dff03cc86bab67ef8106f82ddd1486dcf3cbfef43d556adb5a02201446b5cfc5251e82197b9a4cbe9044e0faa84433bbcbe69b00389d20e3e23a8201483045022100ee9c7504079096d43c66ab70b9fea2f31c5660ede1e9b57a5c26bf976fb883e402200a6a2b7fde1b94ce71767f1908cbf7bae5ea90ec211bba630fd805152d3ec49b01695221038c0246da82d686e4638d8cf60452956518f8b63c020d23387df93d199fc089e82102f1563a8930739b653426380a8297e5f08682cb1e7c881209aa624f821e2684fa2103d2bc85e0b035285add07680695cb561c9b9fbe9cb3a4be4f1f5be2fc1255944c53ae00000000", "P2SH,WITNESS"],
+
+["Remote staking transaction (RSP2WSH)"],
+[[["a9502fbf80b6398542d1dfb7b11e1b913ccaf526a4d8cd8bb96f293656c05fb3", 9, "OP_DUP OP_HASH160 0x14 0xca305850196fe3286cc480fa6e9dd445512755d9 OP_EQUALVERIFY OP_CHECKSIG", 100000000]],
```

Comment:
If I correctly understood the idea of the tests,  the script should be an RSP2WSH script (`0x52 0x14 <staking key hash> 0x20 <spending script hash>`) not P2PKH. And there should be two test cases for each of RSP2WSH and RSP2WPKH: when the transaction type is `COINBASE` and when the transaction type is `REGULAR`.